avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -174,16 +174,28 @@ def splitIniValueToArraySteps(cfgValues):
 
         Returns
         --------
-        items : 1D numpy array
-            values as 1D numpy array
+        items : 1D numpy array or list
+            values as 1D numpy array or list (in the case of strings)
     """
 
     if ':' in cfgValues:
         itemsInput = cfgValues.split(':')
         items = np.linspace(float(itemsInput[0]), float(itemsInput[1]), int(itemsInput[2]))
     else:
         itemsL = cfgValues.split('|')
-        items = np.array(itemsL, dtype=float)
+        flagFloat = False
+        flagString = False
+        for its in itemsL:
+            if its.isalpha():
+                flagString = True
+            if '.' in its:
```

Comment:
good point! however, currently we do not allow a comma as decimal sign, for example if relTh = 1,4 instead of 1.4 this doesn't work, shall we change this behaviour?